GliaWidgets/Component/Button/Action/ActionButtonStyle.swift

```diff
@@ -18,7 +18,7 @@ public struct ActionButtonStyle: Equatable {
     public var backgroundColor: ColorType
 
     /// Corner radius of the button.
-    public var cornerRaidus: CGFloat?
+    public var cornerRaidus: CGFloat
```

Comment:
As far as I understand, you would still need to put the `?` because integrators, for whatever reason, could have here `nil` as an assignment, and with this change, it would be a breaking change.

Reply:
fixed